geomstats/information_geometry/multivariate_normal.py

```diff
@@ -225,6 +351,125 @@ def pdf(x):
         return pdf
 
 
+class MultivariateGeneralNormalDistributions(InformationManifoldMixin, ProductManifold):
```

Comment:
Should this inherit from openset too?

Reply:
> Should this inherit from openset too?

@ninamiolane what would be the benefit ?